src/scenes/ObservationsRouter/details/exportObservationResults.ts

```diff
@@ -185,23 +192,33 @@ function makePlotSpeciesCsv(observationResults: ObservationResults): Blob {
   return makeCsv(columnHeaders, data);
 }
 
-export default function exportObservationResults(params: UseExportObservationResultsParams) {
-  const { observationResults } = params;
-  const prefix = `${observationResults.plantingSiteName}-${observationResults.completedDate}`;
-  const dirName = `${prefix}-${strings.OBSERVATION}`;
-
-  return downloadZipFile({
-    dirName,
-    files: [
-      {
-        fileName: dirName,
-        content: makeObservationCsv(observationResults),
-      },
-      {
-        fileName: `${prefix}-${strings.SPECIES}`,
-        content: makePlotSpeciesCsv(observationResults),
-      },
-    ],
-    suffix: '.csv',
-  });
+export default async function exportObservationResults(params: UseExportObservationResultsParams) {
+  const { observationId, defaultTimeZone, plantingSites, species } = params;
+
+  const resultsWithUnknown = await ObservationsService.getSingleObservationResults(observationId, true);
+  if (resultsWithUnknown.requestSucceeded && resultsWithUnknown.data?.observation) {
```

Comment:
We should probably show an error if the request fails. On the other hand, this code doing nothing is an improvement over the other CSV export buttons in the app, whose reaction to a failed API request is to download a CSV file whose contents are the string `undefined`.